test-vfkit/vfkit_suite_test.go

```diff
@@ -57,7 +57,7 @@ func init() {
 	privateKeyFile = filepath.Join(tmpDir, "id_test")
 	publicKeyFile = privateKeyFile + ".pub"
 	ignFile = filepath.Join(tmpDir, "test.ign")
-	cmdDir = filepath.Join("../cmd")
+	cmdDir = "../cmd"
```

Comment:
Can you add the lint error to the commit log so that it’s easier to understand what the problem was?

Reply:
Yes, done